db/1.13/modules/ROOT/pages/database-connector-connection.adoc

```diff
@@ -61,6 +61,7 @@ In the XML editor, both the spring and database configurations look like this:
 </db:config>
 ----
 
+[[data-source-connection-xa]]
 == Configure a Data Source Reference Connection When in XA Transaction Mode
 
 If your Mule app requires operations bound to XA transactions, and connection pooling is enabled for the database configuration in use, the default connection pool implementation for Data Source Reference Connection is Enhydra XAPool.
```

Comment:
As far as I remember, Enhydra is the underlying datasource. The pooling datasource is the one provided by Bitronix.

Reply:
Enhydra has a XAPool also, but I think we are not using it. We are only using the XA datasource implementations. I'd only mention about datasource implementation.

Reply:
Hi guys @nicomz @acurci-at-mulesoft , I'm a bit lost, so if the user selects Data Source Reference connection, what is the default connection pool then? See the following section and example after that sentence:

_Enhydra XAPool does not support prepared statements. To execute these operations use the Bitronix connection pool. Refer to xref:mule-runtime::using-bitronix-to-manage-transactions.adoc#configuring-the-bitronix-transaction-manager[Configuring the Bitronix Transaction Manager] documentation for details._

_The following example shows how to configure a *Data Source Reference Connection* for a Derby database when in XA transaction mode:
. In Studio, in the `src/main/resources` folder, create a `datasource-config.xml` spring bean file.
. Add the `XADataSource` object to the end of the class `org.enhydra.jdbc.standard.Standard`, for example, `class="org.enhydra.jdbc.standard.StandardXaDataSource`:_